src/SDKs/Authorization/Authorization.Tests/Graph/GraphManagementClient.cs

```diff
@@ -248,11 +248,12 @@ private HttpRequestMessage CreateRequest(string uri, HttpMethod method, string b
 
             if (this.testEnvironment != null && this.testEnvironment.TokenInfo != null)
             {
-                // Not Supported in current code
-                // var tokenCredentials = (TokenCredentials)this.testEnvironment.Credentials;
-                // httpRequest.Headers.Authorization = new AuthenticationHeaderValue(
-                //                                this.testEnvironment.AccessTokenType,
-                //                                this.testEnvironment.AuthorizationContext.AccessToken);
+				// Not Supported in current code
+				// var tokenCredentials = (TokenCredentials)this.testEnvironment.Credentials;
+				httpRequest.Headers.Authorization = new AuthenticationHeaderValue("Bearer",
+					"Put the access token for graph here for the tests to work in record mode");
```

Comment:
earlier the header code was commented out,due to which the headers werent being added to the graph call.Hence this has been added.Also for the tests we need the token with username/password login which isnt supported .